pages/calendar/components/CalendarGroupList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import classNames from "classnames/bind";
+
+import ColorCheckBox from "@/components/ColorCheckBox";
+
+import { GroupColorType } from "@/types/type";
+
+import styles from "./CalendarGroupList.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface GroupType {
+  id: number;
+  groupName: string;
+  groupColor: GroupColorType;
+}
+
+interface CalendarGroupListProps {
+  groupList: GroupType[];
+}
+
+export default function CalendarGroupList({ groupList }: CalendarGroupListProps) {
+  return (
+    <div className={cn("container")}>
+      <p className={cn("enter-group")}>현재 가입된 그룹</p>
+      <ul className={cn("group-container")}>
+        {groupList.map((group) => (
+          <React.Fragment key={group.id}>
+            <ColorCheckBox color={group.groupColor} title={group.groupName} />
+          </React.Fragment>
```

Comment:
```suggestion
          <ColorCheckBox key={group.id} color={group.groupColor} title={group.groupName} />
```
p4: 여기 `Fragment ` 없이 `ColorCheckBox`에 직접 `key`를 부여하면 문제가 생길까요??

Reply:
찾아보니까 지금같을때는 단일 컴포넌트 map이라서 바로쓰는게 더 좋다고 하네요!